src/codegen_test.cpp

```
@@ -450,6 +447,178 @@ void CodeGen::emitPropertyItCall(CallStmt &s) {
+    std::initializer_list<const char*> names)
+{
+    directives.erase(
+        std::remove_if(directives.begin(), directives.end(), [&](const Directive &d) {
+            for (const char *n : names)
+                if (d.name == n) return true;
+            return false;
+        }),
+        directives.end());
+}
+
+void CodeGen::emitItDirective(std::unique_ptr<FnStmt> &s) {
```

Cppcheck (2.20.0)

[style] The function 'emitItDirective' is never used.

(unusedFunction)